### PR TITLE
[instant] Add infura project id via env variables when bundling instant

### PR DESCRIPTION
**Pull request by BMillman19**
## Description

Adds two new env variables to `.env_example`. `INSTANT_INFURA_PROJECT_ID_PRODUCTION` and `INSTANT_INFURA_PROJECT_ID_DEVELOPMENT`

## Testing instructions

Try bundling instant with a local `.env` file

## Types of changes

* New feature (non-breaking change which adds functionality)

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/instant/webpack.config.js [hunk 1]
```diff
@@ -141,24 +147,32 @@ const generateConfig = (dischargeTarget, heapConfigOptions, rollbarConfigOptions
...
-                        options: {
-                            "plugins": ["transform-runtime"],
-                            'presets': [
-                                ['env', {
-                                    'targets': {
-                                        "chrome": 41
-                                    },
-                                }],
-                            ],
+                    use: [
```
Comment (BMillman19) [inline packages/instant/webpack.config.js]: my prettier did this, I guess CI is not looking at `.js` files

File: packages/instant/webpack.config.js [hunk 2]
```diff
@@ -89,11 +89,17 @@ const generateConfig = (dischargeTarget, heapConfigOptions, rollbarConfigOptions
...
+            ? process.env.INSTANT_INFURA_PROJECT_ID_PRODUCTION
+            : process.env.INSTANT_INFURA_PROJECT_ID_DEVELOPMENT;
+
     const envVars = {
         GIT_SHA: JSON.stringify(GIT_SHA),
         NPM_PACKAGE_VERSION: JSON.stringify(process.env.npm_package_version),
         ROLLBAR_ENABLED: rollbarEnabled,
-        HEAP_ENABLED: heapEnabled
+        HEAP_ENABLED: heapEnabled,
+        INSTANT_INFURA_PROJECT_ID: JSON.stringify(infuraProjectId),
```
Comment (PirosB3) [inline packages/instant/webpack.config.js]: is `infuraProjectId` a string? if so, why do we want to serialize it to JSON?

File: packages/instant/src/constants.ts [hunk 3]
```diff
@@ -53,8 +53,8 @@ export const META_MASK_CHROME_STORE_URL =
 export const META_MASK_OPERA_STORE_URL = 'https://addons.opera.com/en/extensions/details/metamask/';
```
Comment (fragosti) [inline packages/instant/src/constants.ts]: Unrelated to this change -- `HOST_DOMAINS_EXTERNAL` seems to be missing 0x.org which is not good :P

Comment (BMillman19) [reply packages/instant/src/constants.ts]: Good catch, will update in this PR

File: packages/instant/webpack.config.js (hunk 2 above)
Comment (BMillman19) [reply packages/instant/webpack.config.js]: `infuraProjectId` is a number. This stringify call is effectively converting from a number to a string. This seems to be the convention we're following in this config file. I'm happy to change it up but we should prob be consistent either way. @fragosti thoughts?

Comment (fragosti) [reply packages/instant/webpack.config.js]: This is a fun little `DefinePlugin` gotcha
https://stackoverflow.com/questions/50454341/why-json-stringifyproduction

**Pull request merged**